src/tests/system/tests/test_ipa.py

```diff
@@ -127,6 +127,95 @@ def test_ipa__hostpublickeys_by_ip(client: Client, ipa: IPA, public_keys: list[s
         assert f"{ip} {key}" in result.stdout_lines, "Did not get expected public keys!"
 
 
+@pytest.mark.ticket(gh=7583)
+@pytest.mark.importance("low")
+@pytest.mark.parametrize("option, output", [(None, "{hostname}:{port} {key}"), ("-o", "{hostname} {key}")])
```

Comment:
`output` is no longer used with the parameterization.

Reply:
updated.

Reply:
Something went wrong and `output` is still declared in the parameterization, although it is not used.

Reply:
Should I remove it?, we decided to keep it for good readability of tests, or please correct me if I misunderstood.

Reply:
Yes, please remove it. It is absolutely useless to have that definition there because it is not used. In addition, there is an `output` variable in use, which drives to confusion.

Reply:
updated.